_benchmark/workloads/index.md

```diff
@@ -0,0 +1,249 @@
+-  `document-count`  The number of documents in the `source-file` that determines which client indexes correlate to which part of the document corpus (each N client get on N-th of the document corpus). When using a source that contains a document with a parent-child relationship, specify the number of parent documents. 
+- `uncompressed-bytes` The size in bytes of the source file after decompression, used to show you much disk space the decompressed source file needs. You can also indicate the number of `compressed-bytes`, the size of the source file before decompression, which can help you assess the time it'll take for the workload to ingest the documents
+
+### Operations
+
+The `operations` element lists the API OpenSearch operations performed by the workload For example, you can set an operation to `create-index`, which creates an index of Benchmarked data after the workload completes. Operations are usually listed inside of `schedule`.
+
+### Schedule
+
+The `schedule` element contains a list of actions and  operations run by the workload. Operations run according to the order in which they appear in the `schedule`. The following example illustrates a `schedule` with multiple operations, each defined by their `operation-type`. 
+
+```json
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": 5000
+      },
+      "warmup-time-period": 120,
+      "clients": 8
+    },
+    {
+      "operation": {
+        "name": "query-match-all",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+According to this schedule, each action will run in the following order.
+
+1. The `create-index` operation creates an index with benchmarked data.
+2. The `cluster-health` operation assess the health of the cluster before running the workload. In this example, the workload will wait until the status of cluster's health is `green`.
+3. The `bulk` operation run the `bulk` API to index `500` documents at once.
+4. Before benchmarking, the workload waits until the specified `warmup-time-period` passes. In this example, the warmup period if `120` seconds.
+5. The `clients` options defines the number of clients that will run the remaining actions in the schedule concurrently.
+6. The `search` runs a `match_all` query to find documents that match each other after they are indexed by the `bulk` API using the 8 clients specified.
```

Comment:
"runs a `match_all` query to find documents that match each other after "
can be clarified with something like
"runs a `match_all` query to match all documents after they have been indexed"